modules/svg/points.js

```diff
@@ -122,6 +123,22 @@ export function svgPoints(projection, context) {
             .append('path')
             .call(markerPath, 'shadow');
 
+        // add a ring if there's a radius/diameter tag
+        enter
+            .filter((d) => getRadiusTag(d.tags))
+            .append('circle')
+            .attr('class', 'radius')
+            .attr('data-nodeId', node => node.id);
+
+        // we need to trigger a re-render when the zoom changes
+        selection
+            .selectAll('.radius')
+            .data([zoom])
+            .attr('r', (_zoom, _zero, [self]) => {
+                const node = graph.entities[self.dataset.nodeId];
+                return getRadiusInPixels(node, context.projection);
```

Comment:
Should we constrain the radius for rendering purposes? What happens in case of a mistake such as a negative or very large value?

Reply:
Negative numbers are handled by us now. For large values, we just get a massive SVG `<circle />` that's too big to see at an editable zoom-level. Not sure if we need an upper limit?